jetty-integrations/jetty-nosql/src/main/java/org/eclipse/jetty/nosql/mongodb/MongoSessionDataStore.java

```diff
@@ -458,8 +466,8 @@ public void initialize(SessionContext context) throws Exception
     public void doStore(String id, SessionData data, long lastSaveTime) throws Exception
     {
         // Form query for upsert
-        final BasicDBObject key = new BasicDBObject(__ID, id);
-
+        //final BasicDBObject key = new BasicDBObject(__ID, id);
```

Comment:
Are we sure that mongo hasn't changed the way it stores documents? In other words, we need to check that a session written out by the older version of mongo is still readable by this new version of mongo.